.gitignore

```diff
@@ -1,2 +1,5 @@
 target/
 **/Cargo.lock
+*/.cargo/config
+.vscode/
+.DS_Store
```

Comment:
These are better kept in a personal `core.excludesFile`: https://git-scm.com/docs/gitignore